### PR TITLE
Update dependencies and fix codesniffer issues with new dependencies

### PR DESCRIPTION
**Pull request by dshanske**
This is just a simple fix to update the codesniffer dependencies.

### CODE REVIEW

File: includes/class-indieauth-authorize.php [hunk 1]
```diff
@@ -191,10 +191,10 @@ public function get_authorization_header() {
 	}
 
 			/**
-	 * Extracts the token from the authorization header or the current request.
-	 *
-	 * @return string|null Token on success, null on failure.
-	 */
+			 * Extracts the token from the authorization header or the current request.
```
Comment (pfefferle) [inline includes/class-indieauth-authorize.php]: Indents: Please correct the alignment of the code block

File: includes/class-indieauth-authorize.php [hunk 2]
```diff
@@ -210,23 +210,23 @@ public function get_provided_token() {
 		return null;
 	}
 			/**
-	 * Extracts the token from the given authorization header.
-	 *
-	 * @param string $header Authorization header.
-	 *
-	 * @return string|null Token on success, null on failure.
-	 */
+			 * Extracts the token from the given authorization header.
```
Comment (pfefferle) [inline includes/class-indieauth-authorize.php]: Indents: Please correct the alignment of the code block

File: includes/class-indieauth-authorize.php [hunk 3]
```diff
@@ -210,23 +210,23 @@ public function get_provided_token() {
...
 			return $matches[1];
 		}
 		return null;
 	}
 			/**
-	 * Extracts the token from the current request.
-	 *
-	 * @return string|null Token on success, null on failure.
-	 */
+			 * Extracts the token from the current request.
```
Comment (pfefferle) [inline includes/class-indieauth-authorize.php]: Indents: Please correct the alignment of the code block

**Review by pfefferle — APPROVED** [review]

File: includes/class-indieauth-authorize.php (hunk 3 above)
Comment (dshanske) [reply includes/class-indieauth-authorize.php]: I don't know why phpcs keeps pulling things out of alignment. It is supposed to identify that. And this PR is all about updating it, so I will be making the rounds of the other plugins

**Pull request merged**